be/src/runtime/stream_load/stream_load_pipe.h

```diff
@@ -86,28 +86,32 @@ class StreamLoadPipe : public MessageBodySink, public FileReader {
     // If _total_length == -1, this should be a Kafka routine load task,
     // just get the next buffer directly from the buffer queue, because one buffer contains a complete piece of data.
     // Otherwise, this should be a stream load task that needs to read the specified amount of data.
-    Status read_one_message(std::unique_ptr<uint8_t[]>* data, size_t* length, size_t padding) override {
+    Status read_one_message(std::unique_ptr<uint8_t[]>* buf, size_t* buf_cap, size_t* buf_sz, size_t padding) override {
```

Comment:
What's the purpose of buf_cap?

Reply:
`buf_cap` denotes the capacity of buf. When `buf_cap` is bigger than payload size, `buf` could be reused and the actual size of payload would be returned in `buf_sz`. When `buf_cap` is smaller than payload size, `buf` should grow to required size and the original one should be released.

To make this clear, more comment is added.